server/sonar-server/src/test/java/org/sonar/server/qualitygate/ws/QualityGatesWsTest.java

```diff
@@ -69,11 +69,19 @@ public void setUp() {
     SelectAction selectAction = new SelectAction(mock(DbClient.class), mock(UserSessionRule.class), mock(ComponentFinder.class));
 
     tester = new WsTester(new QualityGatesWs(
-      new ListAction(qGates), new ShowAction(qGates), new SearchAction(projectFinder),
-      new CreateAction(null, null, null), new CopyAction(qGates), new DestroyAction(qGates), new RenameAction(qGates),
+      new ListAction(qGates),
```

Comment:
this test is doing tests which actually belong to each action's tests
legacy, legacy...